app/assets/stylesheets/sephora_style_guide/partials/product-card.scss

```diff
@@ -0,0 +1,172 @@
+.product-card {
+  width: 100%;
+
+  .product-card-image {
+    width: 100%;
+    display: block;
+  }
+
+  .product-card-actions {
+    position: relative;
+    width: 100%;
+    height: $product-card-actions-height;
+
+    .product-card-add-to-bag,
+    .product-card-add-to-wish-list {
+      margin-top: -$product-card-actions-height;
+
+      @include media-breakpoint-down(lg) {
+        display: none;
+      }
+    }
+  }
+
+  &:hover {
+    .product-card-wish-list-heart {
+      display: block;
+    }
+  }
+
+  .out-of-stock {
+    @include font-avalon-demi;
+    top: 37%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 90%;
+    height: $product-card-actions-height;
+    background-color: rgba(255, 255, 255, 0.75);
+    position: absolute;
+    text-transform: uppercase;
+    color: $grey-dark;
+
+    @include media-breakpoint-down(md) {
+      top: 25%;
+    }
+  }
+
+  .product-card-wish-list-heart {
+    top: 0;
+    right: 15px;
+    width: $product-card-wish-list-heart-width;
+    height: $product-card-wish-list-heart-height;
+    opacity: 0.8;
+    background: url('/assets/sephora_style_guide/heart.svg');
```

Comment:
Instead of loading it as a background using CSS, we use a thing called inline SVG. In your view (Slim) you just need to do:
```slim
= inline_svg 'heart.svg'
```
And it will render the svg inline just like an image